spec/src/main/doc/javaee/injectionandresolution_ee.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[injection_el_resolution_ee]]
+
+== Dependency injection, lookup and EL in Java EE
+
+When running in Java EE, the container must extend the rules defined in <<injection_and_resolution>> and may also provide references to contextual instances by Unified EL expression evaluation.
+
+When resolving a name in an EL expression, the container considers the bean name and selected alternatives.
+
+[[selection_ee]]
+
+=== Modularity in Java EE
+
+In the Java EE module architecture, any Java EE module or library is a module. The Java EE module is a bean archive if it contains a +beans.xml+ file, as defined in <<bean_archive>>.
+
+When running in Java EE, the container must extend the accessibility rules for beans and alternatives defined in <<selection>> to JSP/JSF pages using EL resolution.
```

Comment:
The container should not really extended the accessibility rules. Instead, the container should follow the accessibility rules and make sure that only beans available from injection in the module that defines the JSP/JSF pages are resolved.